bpf/bpf_host.c

```diff
@@ -1126,9 +1123,12 @@ int to_host(struct __ctx_buff *ctx)
 #endif /* ENABLE_HOST_FIREWALL */
 
 out:
-	if (IS_ERR(ret))
+	if (IS_ERR(ret)) 
```

Comment:
Thank you for the new version!

This line seems to add whitespace, which caused this error on one of the checks:

> ERROR:TRAILING_WHITESPACE: trailing whitespace
> #35: FILE: bpf/bpf_host.c:1126:
> +^Iif (IS_ERR(ret)) $

Can you please fix that? (and again, please squash commits)

Reply:
I trying, also I already squashed commit and the count of commit is down to 1 now, thank you for your patient guidance.